### PR TITLE
v2.0

### PR DESCRIPTION
**Pull request by MatsAnd**
* Add ``/me`` endpoints
* Remove user targetting on the ``/students`` endpoint

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "azf-pifu-api",
   "version": "1.0.3",
```
Comment (runely) [inline package.json:3]: Versjonsnummeret burde kanskje hatt en bump?

**Review by runely — CHANGES_REQUESTED** [review]
Se kommentar

File: package.json (hunk 1 above)
Comment (MatsAnd) [reply package.json:3]: ENIG!

**Review by runely — APPROVED** [review]

**Comment by MatsAnd** [comment]
https://github.com/vtfk/azf-pifu-api/releases/tag/2.0.0

**Pull request merged**